x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table.tsx

```diff
@@ -60,14 +73,21 @@ export const SpikeAnalysisTable: FC<SpikeAnalysisTableProps> = ({
     {
       field: 'pValue',
       name: (
-        <>
-          {i18n.translate(
-            'xpack.aiops.correlations.failedTransactions.correlationsTable.logRateLabel',
+        <EuiToolTip
```

Comment:
In other apps I'm used to seeing "?" [icon tips](https://elastic.github.io/eui/#/display/tooltip#icontip) that indicate that there's something to read.